app/helpers/user_helper.rb

```diff
@@ -1,2 +1,17 @@
 module UserHelper
+  def signup
+    visit "/signup"
+    fill_in "Name", with: "Ben"
+    fill_in "Email", with: "ben@gmail.com"
```

Comment:
in tests, it would be nice to replace @gmail.com with @example.com as Sophie suggested

Reply:
Will change it to example in the next merge then :D